solution3.py

```diff
@@ -0,0 +1,18 @@
+bcainfo="""BCA course information:
+Bachelor of computer Application.
+It is a three years undergraduate course."""
+
+with open('BCA.txt','w') as bca:
+    bca.write(bcainfo)

```

Comment:
can you explain this line?

Reply:
The string "It is a three years undergraduate course." will be written to the 'BCA.txt' file in the same directory where the Python script is located. If the file 'BCA.txt' does not exist, it will be created. If it already exists, the previous content will be overwritten with the new content.